mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
to get 25 fps in maya then you need to enter "pal"
maya only accepts strings as input to change fps. and some of the strings are simple like:
> "12fps" => maya set settings to "12fps" and is now working with 12 fps

But for some reasons other strings are like this:
> "pal" => maya set settings to "pal" and is now working with 25 fps

So this wont work:
> "25 fps"  => maya says string is not acceptable

The example above are not actual python commands for maya.